database/src/resources/EnsayoConsultas.java

```diff
@@ -0,0 +1,78 @@
+package resources;
+
+import java.util.Vector;
+import DB_Connection.DBConnection;
+
+public class EnsayoConsultas {
+
+    public static void main (String args[]){
+
+        // Se crea la instancia del manager:
+        DBConnection aQuery = new DBConnection("","","","","","");
+
+        // Se guarda el resultado de la consulta en una arreglo de Objetos
+        // Nota: Siempre va a tener dos posiciones.
+        Object[] objeto= aQuery.read_DB("SELECT * FROM users;");
+
+        // En la posición 0 va a contener el tipo de resultado, solo existen
+        // dos tipos de resultados, "Error" cuando algo falló en el query,
+        // o "QueryResult" cuando la consulta arrojó al menos un registro.
+        String tipo_resultado = (String) objeto[0];
+        System.out.println("\n El tipo de resultado que arrojo la consulta fue un " + tipo_resultado + "\n");
```

Comment:
is this println necessary?

Reply:
why are we using println? i dont think we should show this information.
i not sure about this file
the functionality is ok but you talk about condictionals but i dont see anyone (read line 59 "si queremos acceder") this functionalities should be functions